impexp-client/src/main/java/org/citydb/gui/modules/kml/preferences/GeneralPanel.java

```diff
@@ -464,6 +502,17 @@ private void setEnabledComponents() {
 		exportGltfV2.setEnabled(createGltfCheckbox.isSelected());
 		enableGltfDracoCompression.setEnabled(createGltfCheckbox.isSelected());
```

Comment:
here should be `enableGltfDracoCompression.setEnabled(createGltfCheckbox.isSelected() && exportGltfV2.isSelected());`

Reply:
fixed with 51e8d9c8bac3ac6292911183c419b0575cf727d8

Reply:
This is actually not needed because line 326 already takes care of it in runtime:
```
exportGltfV1.addItemListener(e -> enableGltfDracoCompression.setEnabled(!exportGltfV1.isSelected()));
```

Reply:
@Son-HNguyen, well, without the fix, a situation like shown below is possible:

![image](https://user-images.githubusercontent.com/5517439/101046243-2b649f00-3581-11eb-873b-19a18fabb608.png)

Reply:
Still had issues with this checkbox and hope that d5c74a562df3df06cb6fe3275c34abd519cf5201 now fixes them. Please check :-)